src/main/scala/com/tapad/docker/DockerComposePlugin.scala

```diff
@@ -107,6 +109,26 @@ class DockerComposePluginLocal extends AutoPlugin with ComposeFile with DockerCo
         stopRunningInstances(state, args)
     }
 
+  lazy val dockerComposeRestartCommand = Command.args("dockerComposeRestart", ("dockerComposeRestart", "Restarts a local Docker Compose instance."),
+    s"Supply '$skipPullArg' as a parameter to use local images instead of pulling the latest from the Docker Registry. " +
+      s"Supply '$skipBuildArg' as a parameter to use the current Docker image for the main project instead of building a new one.", "") {
```

Comment:
You should also include:
"Supply the Instance Id to restart a particular instance"